sheeprl/algos/a2c/args.py

```diff
@@ -0,0 +1,33 @@
+from dataclasses import dataclass
```

Comment:
Can you please delete this file?
Thanks

Reply:
My eyes have gone blind! Done!